libs/application/templates/healthcare-work-permit/src/fields/CustomFormConclusionSectionField/index.tsx

```diff
@@ -0,0 +1,148 @@
+import {
+  FieldBaseProps,
+  FieldComponents,
+  FieldTypes,
+} from '@island.is/application/types'
+import { Box, Button, Text, PdfViewer } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { FC, useState } from 'react'
```

Comment:
```suggestion
import { useState } from 'react'
```

Reply:
This import is used in the file

Reply:
Didn't see the comment above related to this, I'll update